inference-engine/src/readers/ir_reader/CMakeLists.txt

```diff
@@ -35,11 +36,6 @@ target_link_libraries(${TARGET_NAME} PRIVATE pugixml)
 # code style
 
 add_cpplint_target(${TARGET_NAME}_cpplint FOR_TARGETS ${TARGET_NAME})
-add_clang_format_target(${TARGET_NAME}_clang_format FOR_TARGETS ${TARGET_NAME})
-
-# developer package
-
-ie_developer_export_targets(${TARGET_NAME})
```

Comment:
@ilyachur Why did you remove this line and why you didn't add the same line to the new target? Just some plugins in separate repositories can use those libraries for own tests/samples and expect that they are built as a part of `ie_developer_all` target.

@ilya-lavrenov FYI

Reply:
IR reader is not a library which should be exported and used directly. Do you mean that it's not built in case if `make ie_dev_targets` is called? It should work anyway, because `ie_dev_targets` depends on MULTI / HETERO plugin, and such plugins depends on readers plugin

Reply:
discussed in skype. PR with the fix https://github.com/openvinotoolkit/openvino/pull/1202